### PR TITLE
Added CONTRIBUTING.md and updated contribution guidelines.

### PR DESCRIPTION
**Pull request by beardyman**

### CODE REVIEW

**Comment by beardyman** [comment]
Closes #29

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+The following is a set of guidelines for contributing to php-sparkpost,
+which is hosted in the [SparkPost Organization](https://github.com/sparkpost) on GitHub.
+These are just guidelines, not rules, use your best judgment and feel free to
+propose changes to this document in a pull request.
+
+## Submitting Issues
+* You can create an issue [here](https://github.com/sparkpost/php-sparkpost/issues/new), but
+  before doing that please read the notes below on debugging and submitting issues,
+  and include as many details as possible with your report.
+* Include the version of python-sparkpost you are using.
```
Comment (richleland) [inline CONTRIBUTING.md]: php-sparkpost

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+* Perform a [cursory search](https://github.com/issues?utf8=%E2%9C%93&q=is%3Aissue+user%3Asparkpost+repo%3Aphp-sparkpost)
```
Comment (beardyman) [inline CONTRIBUTING.md]: It looks like the repo part of this doesn't work right.  I'm looking into it.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+
+### Contribution Steps
+
+#### Guidelines
+
+- Provide documentation for any newly added code.
+- Provide tests for any newly added code.
+- Follow [PSR-1](http://www.php-fig.org/psr/psr-1/)
+
+1. Create a new branch named after the issue you’ll be fixing (include the issue number as the branch name, example: Issue in GH is #8 then the branch name should be ISSUE-8))
```
Comment (richleland) [inline CONTRIBUTING.md]: extra paren at the end

File: README.md [hunk 4]
```diff
@@ -118,20 +118,3 @@ try {
...
-### Setup
-Run `composer install` inside the directory to install dependecies and development tools.
-
-### Testing
-Once all the dependencies are installed, you can execute the unit tests using:
-```
-composer test
-```
-
-### Contributing
```
Comment (richleland) [inline README.md]: Probably worth it to keep a Contributing header and just say `See CONTRIBUTING` where CONTRIBUTING links to the file.

File: CONTRIBUTING.md (hunk 2 above)
Comment (richleland) [inline CONTRIBUTING.md]: https://github.com/SparkPost/php-sparkpost/issues?q=is%3Aissue

**Pull request merged**